dev-util/tartos/tartos-1.0.0~git.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="TAR to shell script"
+DESCRIPTION="The tartos script is a standalone bash shell script which binds \
+different type of TAR archives to a single shell script (bash). This kind of \
+approach is generally adopted to make a single install scipt from an existing \
+tar archives with executable or installable software. It can be used to make \
+a self extracting archive/installer which does some additional jobs like \
+installing softwares, compared to the conventional tar archive files. The \
+tartos script itself is written in bash and thus depends on it and on the \
+package tar."
+HOMEPAGE="https://github.com/neurobin/tartos"
+COPYRIGHT="2015-2017 Md Jahidul Hamid"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="da600454020ed23254c693b037fc1d42fd49318f"
+SOURCE_URI="https://github.com/neurobin/tartos/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="96344d2f5e65ce6429dde3042235907ca3e1dee2b6d71592be3b8eb81d70c6f0"
+SOURCE_FILENAME="tartos-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="tartos-$srcGitRev"
+
+ARCHITECTURES="?x86_gcc2 ?x86_64"
+
+PROVIDES="
+	tartos = $portVersion
+	cmd:tartos
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:shc
+	"
+
+BUILD()
+{
+	shc -f tartos.sh -o tartos
```

Comment:
Is there any good reason to compile this? You could just give the sh file executable permission then install it without the `.sh` suffix

Reply:
Maybe not ... but as it was from the same author I thought I'd give shc a try on this one to see if it could be used in a recipe ...